lib/validators/movie.js

```diff
@@ -3,6 +3,7 @@
 const Joi = require('joi');
 
 module.exports = Joi.object().keys({
-  title: Joi.string().min(1).max(255).required(),
+  title: Joi.string().min(1).max(255),
```

Comment:
hm, so it looks like we don't allow `name` if `title` exists or vice versa, but what happens if we just pass in `title`? there may still be gaps if we're not double writing both the `title` and the `name` column during this step. that is, if we pass in `name`, the `title` value will not be written, meaning that customers would not see the desired output as there are still read dependencies on the `title` column. if we pass in `title`, then we're not writing to the `name` column, meaning there will be gaps when we backfill.

essentially, we want to double write the `name` and `title` columns if one of `title` or `name` are passed in. does that make sense?